src/shared/util.js

```diff
@@ -1531,10 +1532,18 @@ MessageHandler.prototype = {
         });
         break;
       case 'enqueue':
-        this.streamControllers[data.streamId].controller.enqueue(data.chunk);
+        stream = this.streamControllers[data.streamId].controller
+                  ._controlledReadableStream;
```

Comment:
Removed this code in new commit(rather added an extra property `isClosed` in streamsController to check if stream is closed). Also removed unnecessary `let`s in the patch. Thanks for all your suggestions.